### ISSUE TITLE
The executable don't open at all, if internet's cut/down

### ISSUE THREAD

**Issue by Kn0p3XX** [etaylor8086/metroid-prime-randomizer#12]
As the title says.
Is there any chance we'll have a offline version or a starting parameter for offline use? I don't use any of the online stuff anyway.
Thanks.

**Comment by etaylor8086** [comment]
I have version 1.0.0-beta.9 installed, the executable opens fine in airplane mode on my laptop.

![image](https://user-images.githubusercontent.com/5769306/51804550-3036ca00-2217-11e9-8dc9-5ba0d04b6a08.png)

**Comment by Kn0p3XX** [comment]
Here again info stuff, because GitHub peeps should also know what's happening. I talked with BashPrime about this.
I'm using Windows 10 (1809), Metroid Prime Randomizer 1.0.0 beta 9
Anti virus is just Windows Defender, i tried it without exception and with, but makes no difference, though.
I tried the Portable version (ZIP) and the installer, no change.
The executable shows up for a split second in the task manager and goes away, does nothing.
I don't know why this isn't working on my machine.
